lib/iris/fileformats/netcdf/loader.py

```diff
@@ -673,6 +691,8 @@ def __init__(self, var_dim_chunksizes=None):
 
         """
         self.var_dim_chunksizes = var_dim_chunksizes or {}
+        self.Modes = Enum("Modes", ["DEFAULT", "FROM_FILE", "AS_DASK"])
```

Comment:
I like this neater "functional syntax". Unfortunately it seems that not all IDEs can handle it, but are fine with the "class syntax". Given code completion etcetera is a big part of the `Enum` utility, I'd prefer the "class syntax" purely to help IDEs.